confluent_platform/tests/test_e2e.py

```diff
@@ -0,0 +1,38 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+import pytest
+
+from datadog_checks.base.stubs.aggregator import AggregatorStub
+
+from .common import CHECK_CONFIG
+from .metrics import ALWAYS_PRESENT_METRICS, NOT_ALWAYS_PRESENT_METRICS
+
+# TODO: `rate`, `count` metrics are not tested yet in e2e since for the moment those aggregate metrics are not reported.
+# TODO: missing e2e coverage for following metrics. See metrics in metrics.yaml.
+#   - Kafka Connect Task Metrics
+#   - Kafka Connect Sink Metrics
+#   - Kafka Connect Source Tasks Metrics
+#   - Kafka Connect Task Error Metrics
+#   - Confluent Streams Thread
+#   - Confluent Streams Thread Task
+#   - Confluent Stream Processor Node Metrics
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check):
```

Comment:
Remember: by default, functions without signatures aren't type check (so in practice the type hint on `aggregator` has no effect). We should add a signature annotation, or enable [`--check-untyped-defs`](https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-check-untyped-defs).

```suggestion
def test_e2e(dd_agent_check):
    # type: (Any) -> None
```